src/lib/mdns/DiscoveryManager.cpp

```diff
@@ -170,11 +170,11 @@ CHIP_ERROR DiscoveryManager::PublishUnprovisionedDevice(chip::Inet::IPAddressTyp
     char discriminatorBuf[5];  // hex representation of 16-bit discriminator
     char vendorProductBuf[10]; // "FFFF+FFFF"
     // TODO: The text entry will be updated in the spec, update accordingly.
-    TextEntry entries[2] = {
-        { "D", nullptr, 0 },
-        { "VP", nullptr, 0 },
-    };
-
+#if CHIP_ENABLE_ADDITIONAL_ADVERTISING
```

Comment:
```suggestion
#if CHIP_ENABLE_ADDITIONAL_DATA_ADVERTISING
```
and similar for the other places where this is used?

Reply:
Fixed